code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -221,22 +223,35 @@
 			else
 				manual_emote(pick("grooms [p_their()] fur.", "twitches [p_their()] whiskers.", "shakes out [p_their()] coat."))
 
-	//MICE!
+	//MICE! RATS! OH MY!
 	if((src.loc) && isturf(src.loc))
 		if(!stat && !resting && !buckled)
-			for(var/mob/living/simple_animal/mouse/M in view(1,src))
-				if(istype(M, /mob/living/simple_animal/mouse/brown/tom) && inept_hunter)
-					if(COOLDOWN_FINISHED(src, emote_cooldown))
-						visible_message(span_warning("[src] chases [M] around, to no avail!"))
-						step(M, pick(GLOB.cardinals))
-						COOLDOWN_START(src, emote_cooldown, 1 MINUTES)
-					break
+			//Targeting anything in the rat faction nearby
+			for(var/mob/living/M in view(1,src))
 				if(!M.stat && Adjacent(M))
-					manual_emote("splats \the [M]!")
-					M.splat()
-					movement_target = null
-					stop_automated_movement = 0
-					break
+					if (faction_check(M.faction.Copy(), list("rat")))
```

Comment:
Why Copy()?

Also "rat" should be a define here and everywhere its used for typo protection

Reply:
Defines done and reworked the faction checking